lms/envs/common.py

```diff
@@ -1228,6 +1229,7 @@
         for pth in ['courseware', 'histogram', 'navigation']
     ] +
     ['js/' + pth + '.js' for pth in ['ajax-error']] +
+    ['js/bookmarks/main.js'] +
```

Comment:
Are we gonna do js optimization for bookmarks, courseware search feature is being changed and probably we need to change this into factory format as well. `ad16f9f968fd2a0d8a144759c88d8d45d60e38d2`